src/arsenic/session.py

```diff
@@ -315,6 +315,12 @@ async def switch_to_window(self, handle):
             url="/window", method="POST", data={"handle": handle, "name": handle}
         )
 
+    async def new_window(self, window_type: WindowType = None):
+        window_type = window_type if window_type is not None else WindowType.tab.value
+        return await self._request(
+            url="/window/new", method="POST", data={"type": window_type}
```

Comment:
How about this instead?

```suggestion
    async def new_window(self, window_type: WindowType = WindowType.tab):
        return await self._request(
            url="/window/new", method="POST", data={"type": window_type.value}
```

Reply:
Okay, why not.